src/CashFlows.py

```diff
@@ -1110,14 +1119,18 @@ def __init__(self, **kwargs):
     self._inflation = True
     self._yearlyCashflow = None
 
-  def initParams(self, lifetime):
+  def initParams(self, lifetime, pyomoVar=None):
     """
       Initialize some parameters
       @ In, lifetime, int, the given life time
+      @ In, pyomoVar, pyo.ConcreteModel(), concrete model including all variables
```

Comment:
Do we actually want the ConcreteModel instance passed in, or should this be a boolean flag? Also, need to add "optional" after the type entry in this docstring.

Reply:
Yes, it should be boolean now. The previous object array creation was by John's method and it required the initial concrete model. Now that it does not, it can be False/True boolean style.